### PR TITLE
Issue 6505: Tweaking the use of the cluster name property in the Admin CLI

### PR DESCRIPTION
**Pull request by kuldeepk3**
Signed-off-by: Kuldeep Kumar <kuldeep.kumar3@dell.com>

**Change log description**  
This PR allows a consistent use of pravegaservice.cluster.name by Admin CLI cluster commands and other commands (ie, bk list)

**Purpose of the change**  
Fixes https://github.com/pravega/pravega/issues/6505

**What the code does**  
Remove the hardcoded cluster namespace prefix, making it fully overridable via pravegaservice.cluster.name

**How to verify it**  
The following affected commands are working as expected.
In order to see the expected result user need to set `pravegaservice.clusterName`to `pravega/<clustername>`

    cluster get-host-by-container
    cluster list-instances
    cluster list-containers

### CODE REVIEW

File: cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java [hunk 1]
```diff
@@ -194,7 +192,7 @@ private void createZKClient(String zkURL, String clusterName) throws ZKConnectio
         zkClient = CuratorFrameworkFactory
                 .builder()
                 .connectString(zkURL)
-                .namespace(BASE_NAMESPACE + SEPARATOR + clusterName)
+                .namespace(clusterName)
```
Comment (eolivelli) [inline cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java:195]: This doesn't look like a compatible change.

**Review by RaulGracia — CHANGES_REQUESTED** [review]
Several points on this PR:
- Add proper issue title ("Issue (number): Title of original issue")
- Tests are failing
- Once you get the tests to work locally, deploy Pravega on a cluster and show in the PR how the CLI works with this change.

**Review by anirudhkovuru — CHANGES_REQUESTED** [review]
So the issue is to make sure that `cluster` commands can run when the cluster name is provided as `pravega/xxx`. Currently the `cluster` commands require the cluster name to be `xxx` in order to run.

Please make sure to make the configurations in the tests to reflect this change.

File: cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java (hunk 1 above)
Comment (kuldeepk3) [reply cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java:195]: The hard coded value is making things inconsistent and hence we get exceptions. Removing it solves the problem.

**Pull request merged**